### ISSUE TITLE
Gradio 5 ChatInterface: internationalize popup text for retry, undo, and clear buttons

### ISSUE THREAD

**Issue by alexeygridnev** [gradio-app/gradio#10020]
- [V] I have searched to see if a similar issue already exists.

**Is your feature request related to a problem? Please describe.**  
Retry, Undo and Clear buttons are now parts of the Chatbot component of ChatInterface, and there does not seem to be a way to customize the text which is shown when you move your mouse to the button (it says "Retry", "Undo" and "Clear" for the respective buttons). For fully multilingual interfaces, it is useful to be able to change this text.

**Describe the solution you'd like**  
Add parameters "submit_btn", "undo_btn" and "clear_btn" to ChatInterface where you would be able to indicate the popup text you want to see on moving the mouse to the element .

**Additional context**
Might also be useful to be able set those parameters above to None (like "submit_btn" and "stop_btn" parameters), which is useful if a user doesn't need some of this buttons but doesn't want to construct the chat interface completely from scratch.

**Comment by Scottan** [comment]
I'd like to upvote this one (I came on here to raise an issue but found that this one basically says what I wanted). 

I'm building an app with a custom API and we don't have the endpoints for the undo or retry functionality yet, which means clicking on these buttons causes errors. I wanted to hide them from the user, but found the undo_btn and retry_btn parameters have been removed in version 5
https://github.com/gradio-app/gradio/issues/9463

I was able to hide them with some custom CSS:
```
css = """
button[aria-label='Retry'], button[aria-label='Undo'] {
    display: none !important;
}
"""

with gr.Blocks(title=app_title, css=css) as demo:
    ...
```

However, being able to do `undo_btn=None, retry_btn=None` in either the Chatbot or ChatInterface objects would have saved a lot of time and been a lot cleaner.

**Comment by abidlabs** [comment]
cc @freddyaboulton maybe we can add these labels to the i18n dictionaries as well

**Comment by freddyaboulton** [comment]
Yes good idea and we can definitely add to the i18n. Should we also add parameters to the chatbot so users can provide their own text and take precedence over the built-in i18n?

**Comment by abidlabs** [comment]
I think that would add too many parameter specifically to chatbot but @hannahblair will work on a comprehensive solution as part of https://github.com/gradio-app/gradio/issues/2465

**Comment by freddyaboulton** [comment]
Cool!
